mempool/src/shared_mempool/priority.rs

```diff
@@ -261,8 +262,12 @@ impl PrioritizedPeersState {
                     .as_secs()
             });
 
-        let average_mempool_traffic_observed = num_txns_received_since_peers_updated as f64
+        let average_mempool_traffic_observed = num_mempool_txns_received_since_peers_updated as f64
             / max(1, time_elapsed_since_last_update) as f64;
+        let average_committed_traffic_observed = num_committed_txns_received_since_peers_updated
+            as f64
+            / max(1, time_elapsed_since_last_update) as f64;
```

Comment:
add a comment here that this is measured time vs actual onchain time, and so if node is state syncing or lagging might not be precise. and that it can be fixed if needed.

Reply:
Done.